contracts/Super1155.sol

```diff
@@ -368,7 +291,7 @@ contract Super1155 is PermitControl, ERC165Storage, IERC1155, IERC1155MetadataUR
   */
   function uri(uint256 id) external override view returns (string memory) {
```

Comment:
`Super1155` doesn't need to perform all of this string templating on-chain the way we want our ERC-721 implementation to. It is acceptable to directly return the URI with `{id}` in it; the 1155 standard dictates that substitution like this be performed by the client.